src/common/mClockPriorityQueue.h

```diff
@@ -344,6 +351,32 @@ namespace ceph {
       return std::move(*(retn.request));
     }
 
+    Retn dequeue_distributed() {
+      assert(!empty());
+      dmc::PhaseType resp_params = dmc::PhaseType();
+
+      if (!(high_queue.empty())) {
```

Comment:
nit, remove the parenthesis around `high_queue.empty()`.

Reply:
Also deleted it.